### ISSUE TITLE
Pairing fails when providing custom userid and key

### ISSUE THREAD

**Issue by Creat1veM1nd** [albertogeniola/ha-meross-local-broker#36]
Hello,

I am trying to switch from meross app (and cloud) to a local integration. When trying to pair a smartplug mss310 via custom pairer app, it does not pair when providing custom userid and key. If I do not provide it, the pairing works.

What I am doing wrong ?

Thanks for any help!

**Comment by albertogeniola** [comment]
Hi @Creat1veM1nd ,

the usage of custom `userid` and `key` attributes is reserved when pairing against third-party brokers, which might require specific configurations. When using the local-addon you don't need to customize those attributes. 

Are you pairing against the local-addon or against other systems?

**Comment by Creat1veM1nd** [comment]
I paired against a local-addon. But I did not got it working, the problem might be on my end aka misunderstanding the mechanism behind it, I guess. I am using now another integration for my meross because of lack of time. 

Since no one else seems to have this issue (at least I did not find anything related), I  guess it can be closed for now. When finding time to dig into it again I will open up a new one, if stumbling across it again. (I am still learning all that stuff,). 

I hope, this is ok and thanks for your time!
